common-v2/all-prefix.js

```diff
@@ -2,13 +2,27 @@
 // Prevent jQueryBE to be register in AMD
 // which cause conflict when client site use RequireJS
 if (typeof define === "function" && define.amd) {
-    window.tmpDefine = define; 
+    window.tmpDefine = define;
     define = undefined;
 }
 
 (function () {
-    if (document.all && !document.addEventListener) {
-        console.log('[PingOne] IE8 and below are not supported!');
-        return;
+    if (typeof document.documentMode === 'number') {
```

Comment:
document.documentMode is only available on IE. So it is reliable to use it to check if the user is using IE.